templates/expo-template-bare-minimum/android/gitignore

```diff
@@ -0,0 +1,21 @@
+# OSX
+#
+.DS_Store
+
+# Android/IntelliJ
+#
+build/
+.idea
+.gradle
+local.properties
+*.iml
+*.hprof
+
+# BUCK
+buck-out/
+\.buckd/
+*.keystore
+!debug.keystore
+
+# Bundle artifacts
+*.jsbundle
```

Comment:
Are bundles generated in the ios and android folders?

Reply:
I don't think they're bundled anywhere in the app directory, potentially a legacy artifact. You can potentially set the bundle output to inside the native folder, but generally this would only be done for detox which spawns in `ios/build` (already ignored).

Reply:
Validated that it was added in `react-native@0.60` [here](https://github.com/facebook/react-native/pull/17888). Can't determine if some flows still include this value so I'm going to keep it for now.

Reply:
Sounds good, at least we did a sanity check on it now ;)